packages/components/src/styles/prose/prose.stories.ts

```diff
@@ -9,720 +9,190 @@ const { generateTemplate } = storybookTemplate('sd-prose');
-            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
-            Facilisis commodo integer hendrerit tortor.
-          </p>
-          <h4>H4 Nisi eu excepteur anim esse</h4>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
-            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
-            Facilisis commodo integer hendrerit tortor.
-          </p>
-          <h5>H5 Nisi eu excepteur anim esse</h5>
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
-            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
-            Facilisis commodo integer hendrerit tortor.
-          </p>
-        </div>
+          </ul>
+        </li>
+        <li>Unordered list level 1</li>
+        <li>Unordered list level 1</li>
+      </ul>
+      <figure class="sd-media p-4">
+        <img
+          src="./placeholders/images/generic.jpg"
+          alt="A generic placeholder jpg"
+          class="aspect-video object-cover"
+        />
+        <figcaption>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula</figcaption>
+      </figure>
+      <table class="sd-table" style="width: 100%">
+        <thead>
+          <tr>
+            <th class="sd-table-cell">Header</th>
+            <th class="sd-table-cell">Header</th>
+            <th class="sd-table-cell">Header</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          <tr>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+          </tr>
+          <tr>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+          </tr>
+          <tr>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+          </tr>
+          <tr>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+          </tr>
+          <tr>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+            <td class="sd-table-cell">Cell Content</td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  `
+};
+
+/**
+ * Use the `&--full-width` class to make the prose full width.
+ */
+export const FullWidth = {
+  render: () =>
+    html` <div class="sd-prose mb-8">
```

Comment:
Please remove `mb-8` and add a div around both sd-prose divs with the classes `flex flex-col space-y-8`

Reply:
Updated